modules/cnf-image-based-upgrade.adoc

```diff
@@ -0,0 +1,102 @@
+[id="ztp-image-based-upgrade-concept-stages_{context}"]
+= Stages of the image-based upgrade
+
+After generating the seed image on the seed cluster, you can move through the stages on the target cluster by setting the `spec.stage` field to the following values in the `ImageBasedUpgrade` CR:
+
+* `Idle`
+* `Prep`
+* `Upgrade`
+* `Rollback` (Optional)
+
+image::../images/ibu-stages.png[Stages of the image-based upgrade]
+
+[id="ztp-image-based-upgrade-concept-idle_{context}"]
+== Idle stage
+
+The {lcao} creates an `ImageBasedUpgrade` CR set to `stage: Idle` when the Operator is first deployed.
+This is the default stage, there is no ongoing upgrade and the cluster is ready to move to the `Prep` stage.
+
+image::../images/ibu-idle-prep.png[Transition from Idle stage]
+
+You also move to the `Idle` stage to do one of the following steps:
+
+* Finalize a successful upgrade
+* Finalize a rollback
+* Cancel an ongoing upgrade until the pre-pivot phase in the `Upgrade` stage
+
+image::../images/ibu-idle.png[Transitions to Idle stage]
+
+Moving to the `Idle` stage ensures that the {lcao} cleans up resources, so the cluster is ready for upgrades again.
+
+[WARNING]
+====
+If you move to the `Idle` stage after a rollback, the {lcao} cleans up resources that can be used to troubleshoot a failed upgrade.
+====
+
+[id="ztp-image-based-upgrade-concept-prep_{context}"]
+== Prep stage
+
+[NOTE]
+====
+You can complete this stage before a scheduled maintenance window.
+====
+
+For the `Prep` stage, you specify the following upgrade details in the `ImageBasedUpgrade` CR:
+
+* seed image to use
+* resources to back up
+* extra manifests to apply after the upgrade, if any
+* custom catalog sources to retain after the upgrade, if any
+
+Then, based on what you specify, the {lcao} prepares for the upgrade without impacting the current running version.
+During this stage, the {lcao} ensures that the target cluster is ready to proceed to the `Upgrade` stage by checking if it meets certain conditions and pulls the seed image to the target cluster with additional container images specified in the seed image.
+
+You also prepare backup resources with the OADP Operator's `Backup` and `Restore` CRs.
+These CRs are used in the `Upgrade` stage to reconfigure the cluster, register the cluster with {rh-rhacm}, and restore application artifacts.
+
+Additionally to the OADP Operator, the {lcao} uses the `ostree` versioning system to create a backup, which allows complete cluster reconfiguration after both upgrade and rollback.
+
+Once the `Prep` stage finishes, you can cancel the upgrade process by moving to the `Idle` stage or you can start the upgrade by moving to the `Upgrade` stage in the `ImageBasedUpgrade` CR.
+If you cancel the upgrade, the Operator performs cleanup operations.
+
+image::../images/ibu-prep.png[Transition from Prep stage]
+
+[id="ztp-image-based-upgrade-concept-upgrade_{context}"]
+== Upgrade stage
+
+The `Upgrade` stage consists of three phases:
```

Comment:
```suggestion
The `Upgrade` stage consists of two phases:
```
The reason for change - we are talking about only two phases: pre-pivot and post-pivot underneath.